### PR TITLE
Updating Max-Forwards Header Summary

### PR DESCRIPTION
**Pull request by bhaveshraj336**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/http/headers/index.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ Network client hints allow a server to choose what information is sent based on
 - {{HTTPHeader("Expect")}}
   - : Indicates expectations that need to be fulfilled by the server to properly handle the request.
 - {{HTTPHeader("Max-Forwards")}}
-  - : TBD
+  - : Indicates Maximum number of Forwards between Client and Server.
```
Comment (teoli2003) [inline files/en-us/web/http/headers/index.md:171]:
```suggestion
  - : When using [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE), indicates the maximum number of hops the request can do before being reflected back to the sender..
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/http/headers/index.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ Network client hints allow a server to choose what information is sent based on
 - {{HTTPHeader("Expect")}}
   - : Indicates expectations that need to be fulfilled by the server to properly handle the request.
 - {{HTTPHeader("Max-Forwards")}}
-  - : TBD
+  - : When using [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE), indicates the maximum number of hops the request can do before being reflected back to the sender..
```
Comment (teoli2003) [inline files/en-us/web/http/headers/index.md:171]:
```suggestion
  - : When using [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE), indicates the maximum number of hops the request can do before being reflected to the sender.
```

**Review by teoli2003 — APPROVED** [review]

**Comment by bhaveshraj336** [comment]
@teoli2003 I have one suggestion can mdn have security related docs too I would love to work on that ?

**Comment by teoli2003** [comment]
> @teoli2003 I have one suggestion can mdn have security related docs too I would love to work on that ?

See https://developer.mozilla.org/en-US/docs/Web/Security

Note that these docs are really hard to work on. Security is difficult because it is complex and every error has consequences.

I would advise getting used to MDN first, by doing a few other PRs, then getting acquainted with such a corpus by finding errors in them and fixing them.

**Pull request merged**